### DISCUSSION TITLE
Aufteilung der Emissionen nach Sektoren?

### DISCUSSION THREAD

**Discussion by lhm** [codeformuenster/klimawatch#125] [General]
Sollte es bei Klimawatch eine präferierte Aufteilung der Emissionen geben? Die Untergliederung der Emissionen ist zur Zeit etwas uneinheitlich: teilweise gibt es die Aufteilung Strom/Wärme/Verkehr, und anderseits nach Verbrauchssektor (Haushalte, Industrie, GHD, Verkehr). Für die Vergleichbarkeit, und auch um die Zuarbeit einfacher beschreibbar zu machen, wäre es meiner Meinung von Vorteil wenn wir hier versuchen etwas zu vereinheitlichen.

Persönlich würde ich zu der Aufteilung nach Sektoren tendieren, wie sie im [BISKO Standard (Kapitel 4.2)](https://www.ifeu.de/wp-content/uploads/BISKO_Methodenpapier_kurz_ifeu_Nov19.pdf) vorgesehen ist. Da mehr und mehr Kommunen nach diesem Standard bilanzieren, sollte das Hinzufügen und Aktualisieren von Daten dann einfacher sein. Auch für eine eventuelle Integration mit Klimaschutzplaner oder EcoSpeed wäre das vermutlich hilfreich.

Es gibt vermutlich in einzelnen Fällen das Problem, das diese Daten in der einen oder anderen Form nicht vorliegen (z.B. Landau), und dann wäre es vermutlich ok damit zu arbeiten was man hat. Aber falls beide Aufteilungen vorliegen wäre es gut eine Empfehlung für eine Art zu geben.

**Comment by lhm** [comment]
Zum Vergleich:
* [Umweltbundesamt (Bund)](https://www.umweltbundesamt.de/indikator-emission-von-treibhausgasen)
* [LAK Energiebilanzen (Länder)](https://www.lak-energiebilanzen.de/ergebnisse-des-datenabrufs/?a=c600&j=2017&l=1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16&v=anm,insgesamt,agg_vg,agg_verkehr_insgesamt,agg_verkehr_schiene,agg_verkehr_strasse,agg_verkehr_luft,agg_verkehr_wasser,agg_hghd,Stand)
* [Bericht für Halle bei klimaschutz-planer](https://www.klimaschutz-planer.de/ajax.php?action=newWindow&pageID=bilanz_bericht_ausgabe&eparam=commune%3D150020000000%26year%3D2015)

**Comment by silberzwiebel** [comment]
Ich würde hier eher keine Vorgabe machen wollen, um jedwede Einreichung so einfach wie möglich zu machen. Wichtig ist, dass es pro Kommune konsistent ist. Den interkommunalen Vergleich finde ich schwierig und auch nicht unbedingt erstrebenswert. Aber wir können auch gerne eine Empfehlung mitgeben.
